Classes/Infrastructure/Healthcheck/CrHealthcheck.php

```diff
@@ -15,7 +15,7 @@
 class CrHealthcheck implements HealthcheckInterface
 {
     public function __construct(
-        private DbalClientInterface $dbalClient,
+        private Connection           $dbalConnection,
```

Comment:
Just a tiny cosmetic fix
```suggestion
        private Connection $dbalConnection,
```